frontend/src/pages/Sessions/SessionsFeedV3/hooks/useSessionFeedConfiguration.ts

```diff
@@ -23,6 +27,17 @@ export const useSessionFeedConfiguration = () => {
 		`${LOCAL_STORAGE_KEY_PREFIX}SortOrder`,
 		'Descending',
 	)
+	const [resultFormat, setResultFormat] =
+		useLocalStorage<SESSION_FEED_RESULT_FORMAT>(
+			`${LOCAL_STORAGE_KEY_PREFIX}ResultFormat`,
+			'Count',
+		)
+
+	useEffect(() => {
+		if (sessionResultsVerbose) {
+			setResultFormat('Count/Length/ActiveLength')
+		}
```

Comment:
Not sure how big a deal this is unless we need to roll back, but turning off the toggle still uses the updated result format for any users that have it saved in local storage since we don't reset it